handsontable/src/plugins/exportFile/types/csv.js

```diff
@@ -53,10 +62,18 @@ class Csv extends BaseType {
       if (index > 0) {
         result += options.rowDelimiter;
       }
+
       if (hasRowHeaders) {
-        result += this._escapeCell(rowHeaders[index]) + options.columnDelimiter;
+        result = result
+          + this._escapeCell(rowHeaders[index], { sanitizeValue: options.sanitizeValues })
+          + options.columnDelimiter;
```

Comment:
seems like `+` is the slowest method of concating 
either use [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat) or `Array.join` or `+=`  

reference 
1. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition 
> You are advised to not use "" + x to perform [string coercion](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#string_coercion). 
2. https://www.sitepoint.com/javascript-fast-string-concatenation/

Reply:
I changed it to `+=`

Reply:
Btw, the second link states that

> If you’re only joining a few strings, you should use whichever method is most practical. Performance gains or losses will be negligible in all browsers.